package.json

```diff
@@ -6,12 +6,18 @@
     "moment": "^2.22.1",
     "react": "^16.3.2",
     "react-dom": "^16.3.2",
-    "react-scripts": "1.1.4"
+    "react-scripts": "^2.1.1"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
-  }
+  },
+  "browserslist": [
+    ">0.2%",
+    "not dead",
```

Comment:
Any idea what this stuff is?

Reply:
Looks like an explicit list of which browsers we support. Doesn't seem to affect anything on our setup.